lib/spack/llnl/util/tty/log.py

```diff
@@ -440,9 +440,9 @@ def __init__(self, file_like=None, echo=False, debug=0, buffer=False,
         Args:
             file_like (str or stream): open file object or name of file where
                 output should be logged
-            echo (bool): whether to echo output in addition to logging it
-            debug (int): positive to enable tty debug mode during logging
-            buffer (bool): pass buffer=True to skip unbuffering output; note
+            echo     (bool): whether to echo output in addition to logging it
+            debug    (int): positive to enable tty debug mode during logging
+            buffer   (bool): pass buffer=True to skip unbuffering output; note
```

Comment:
Can you remove the whitespace changes that are not related to this PR?

Reply:
yes, sorry about that! I was originally going to update the log.py to do something with phases, but then decided that was the wrong level to do it.